docs-kits/kits/Quality-Kit/page_adoption-view.md

```diff
@@ -421,22 +239,23 @@ To realize the **Business Logic** described in the Quality Kit
 
 all steps of the **Business Process** (described in the [Development View](https://confluence.catena-x.net/display/BDPQ/WORK+IN+PROGRESS+-+Quality+KIT+-+Development+View)), like data provisioning and consuming by the involved partner companies, are realized in compliance with the [Catena-X Data Governance Framework](https://catena-x.net/en/catena-x-introduce-implement/governance-framework-for-data-space-operations).  Under this link you can find the latest version of the framework regulations as download. The documents are seperated in the following levels:
 
-![quality kit data offering diagram](@site/static/img/quality-kit_data-offering-min.png)
-
-<!-- BROKEN LINK............... -->
 **Data Space Level**: 10 Golden Rules of Catena-X
 
-**Use Case Level**: Quality Management specific policy as download: <a target="_blank" href={ require("/office/20230710_Catena-X_UseCasePolicy_Quality_3.0_EN.docx").default } download>20230710_Catena-X_UseCasePolicy_Quality_3.0_EN.docx</a>
+
+**Use Case Level**: [Quality Management specific policy](https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_Quality.pdf)![image](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/148753489/3d45270a-3045-41b7-a187-8ce9789f497c)
+as download.
 
 **Data Offering** and **Usage Level** are defined by bi-lateral aligned policies and contracts between the cooperating partner companies. Content is currently in definition.
 
 ## Standards 
 
 Our relevant standards can be downloaded from the official [Catena-X Standard Library:](https://catena-x.net/de/standard-library)
 
+- [CX - 0018 Eclipse Data Space Connector EDC](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_September23/CX-0018-EclipseDataConnector_EDC_-v.2.0.1.pdf))
+  
 - [CX - 0036 Semantic Model Quality Task](https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_PDF_Maerz/PLM_Quality_Use_Case_Live_Quality_Loops/CX_-_0036_Semantic_Model_Quality_Task_v_1.0.1.pdf)
```

Comment:
Also here

Reply:
@FaGru3n 
The links go to the standard library - this is also used in other KITS like Traceability 
https://eclipse-tractusx.github.io/docs-kits/next/kits/Traceability%20Kit/Architecture%20View%20Traceability%20Kit#standards
Resolved ?